Dockerfile

```diff
@@ -32,4 +32,7 @@ RUN (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pac
 COPY --from=build-env /go/bin/cirunner .
 COPY ./ssh-config /root/.ssh/config
 RUN chmod 644 /root/.ssh/config
-ENTRYPOINT ["./cirunner"]
+
+# changing anything here will break current functionalities of logging and failure handling
+ENTRYPOINT ["/bin/sh", "-c", "set -o pipefail; ./cirunner 2>&1 | tee main.log"]
```

Comment:
why can't we execute this in a script file and put inAppLogging check there as well to prevent main.log file creation where it is not required ??